src/components/BookCard.js

```diff
@@ -15,22 +15,28 @@ const useStyles = makeStyles({
 	},
 	media: {
 		height: 240
+	},
+	rating: {
+		paddingLeft: 15
+	},
+	typography: {
+		paddingLeft: 5
 	}
 });
 
-function BookCard(props) {
+function BookCard(thumbnail, rating, book_title, author) {
```

Comment:
It needs to be `({thumbnail, rating, book_title, author})` and not `(thumbnail, rating, book_title, author)`.

What the first code means is you are taking the props and refactoring it.
```js
function BookCard(props) {
    const {thumbnail, rating, book_title, author} = props
}
```

```js
function BookCard({thumbnail, rating, book_title, author}) {
    
}
```

The above two code blocks mean the same thing. I hope it is clearer now? Please make these changes and create another commit.
Thank You

Reply:
Yes this makes sense. I will change it right away.

Reply:
@mihirgupta0900 changes made. Let me know if it looks good.